dockerfiles/e2e/build.sh

```diff
@@ -20,7 +20,7 @@ fi
 echo "Copying source code ${E2E_DIR} --> ${LOCAL_E2E_DIR}"
 cp -r "${E2E_DIR}" "${LOCAL_E2E_DIR}"
 
-init --name:e2e "$@"
+init --name:e2e --no-cache "$@"
```

Comment:
I think there is something wrong here. Why no-cache should be mandatory if not given in build args ?

Reply:
@benoitf This is not common "build.sh", this is "build.sh" of e2e tests image, and for this image, we exactly need "--no-cache" option.

Reply:
There is something wrong in the Dockerfile if it should be default. I'm -1

Reply:
@benoitf The thing is, that we are installing latest chrome from official google chrome repo in the image ([here](https://github.com/eclipse/che/blob/master/dockerfiles/e2e/Dockerfile#L17)), but as the dockerfile is not changing and the build is happening everytime on the same machine, cache is used and we are stuck with old version of browser.
I'm not aware of any other way of solving this. If you have better idea how to solve this, we're all ears!

Reply:
Even in official docker best practices is this section in one of the examples:
```
# Install tools required to build the project
# We will need to run `docker build --no-cache .` to update those dependencies
RUN apk add --no-cache git
RUN go get github.com/golang/dep/cmd/dep
```

Reply:
@rhopp 
to sum up: you want to invalidate the cache *only* if chome is updated

The approach for this is to use an ADD instruction to a link that is updated when google chrome is updated

Then you should (maybe link is not correct) use instruction like
#invalidate cache
```
ADD http://dl.google.com/linux/chrome/rpm/stable/x86_64/repodata/filelists.xml.gz /tmp/filelists.xml.gz
```
before the yum install instructions

so it means that if the google chrome is not updated then we keep the layers, if google chrome is updated then layers are automatically invalidated

Reply:
This is what we're doing in some Dockerfile to only recompile a dependency if github project has new commits since the last time we ran `docker build`

Reply:
Sounds good. Will try that! Thanks!